src/carl/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@
 # include util in library.
 add_subdirectory(util)
 
+include(GNUInstallDirs)
```

Comment:
Must be included before `$<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}/carl>`, otherwise we end up with an absolute path here